test/unit/selector.js

```diff
@@ -1137,14 +1137,14 @@ QUnit.test( "pseudo - :not", function( assert ) {
 	assert.t( ":not() failing interior", "#qunit-fixture p:not(.foo)", [ "firstp", "ap", "sndp", "en", "sap", "first" ] );
 	assert.t( ":not() failing interior", "#qunit-fixture p:not(#blargh)", [ "firstp", "ap", "sndp", "en", "sap", "first" ] );
 
-	if ( QUnit.jQuerySelectors || this.safari ) {
+	if ( QUnit.jQuerySelectors || !QUnit.isIE ) {
 		assert.t( ":not() failing interior", "#qunit-fixture p:not(div.foo)", [ "firstp", "ap", "sndp", "en", "sap", "first" ] );
 		assert.t( ":not() failing interior", "#qunit-fixture p:not(p.foo)", [ "firstp", "ap", "sndp", "en", "sap", "first" ] );
 		assert.t( ":not() failing interior", "#qunit-fixture p:not(div#blargh)", [ "firstp", "ap", "sndp", "en", "sap", "first" ] );
 		assert.t( ":not() failing interior", "#qunit-fixture p:not(p#blargh)", [ "firstp", "ap", "sndp", "en", "sap", "first" ] );
 	} else {
-		// Support: Chrome 75+, Firefox 67+
-		// Chrome/Firefox don't support `:not(complex selector)`.
+		// Support: IE 11+
```

Comment:
IE 11 was the last IE, I think!

Reply:
Yes, but our syntax allows either `11+` or `11 only` and the team decided the former is better, just in case there is a newer version one day. The `only` versions are mostly when we verified a newer version fixes the issue which cannot be done here.

Reply:
Ah, I didn't realize there was a technical rule that parses these. np, not worth making more complicated then :)

Reply:
We don't have any tooling but we have a grammar defined by @gibson042 in https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-69379197 with my amendment in https://github.com/jquery/contribute.jquery.org/issues/95#issuecomment-448998379.